src/burnchains/mod.rs

```diff
@@ -308,6 +308,14 @@ impl PoxConstants {
     pub fn mainnet_default() -> PoxConstants {
         PoxConstants::new(1000, 240, 192, 25)
     }
+
+    pub fn testnet_default() -> PoxConstants {
+    PoxConstants::new(
+        120, 
+        30, 
+        20,
+        3333333333333333) // total liquid supply is 40000000000000000 µSTX
```

Comment:
So, 8.3% of liquid uSTX can disable PoX?

Reply:
Am I misunderstanding https://github.com/blockstack/stacks-blockchain/issues/1882?

Reply:
Ha, I think 1/120th of liquid supply is the minimum for stacking. What should we use for `pox_rejection_fraction` ?
cc @jcnelson @kantai ?

Reply:
8.3% is fine by me -- I'm in favor of a low threshold so the community can easily test it out.  I was just curious if there was a particular reason for the precise number.

Reply:
Yep, 8.3% works for me -- if someone griefs the PoX anchor blocks, we can always bump it up